### DISCUSSION TITLE
Public peer re-announcement for v0.5

### DISCUSSION THREAD

**Discussion by d47081** [yggdrasil-network/yggdrasil-go#1074] [Q&A] [answered]
Congrats to community with new release!

I have upgraded public peer node, should I send PR somewhere to announce it for v0.5 list?

proto/host/port is same, it's marked offline in v0.4 now but not listed in v0.5 yet

**Comment by neilalexander** [comment, marked as answer]
If you've upgraded a node in-place and the address/port haven't changed then it will appear on the new list on the next run :-)

**Reply by d47081** [reply]
Well so, 

thank you for your work!
